gradle.properties

```diff
@@ -1,7 +1,7 @@
 # Main properties
 group=com.truelayer
 archivesBaseName=truelayer-java
-version=14.3.0
+version=15.0.0
```

Comment:
This is a breaking change because the currency code returned by the GetPayment by id utility is now modelled with an enum, as opposed to a string.